rules/windows/process_creation/proc_creation_win_certutil_decode_suspicious_extensions.yml

```diff
@@ -0,0 +1,90 @@
+title: Suspicious Certutil Decoding
+id: 769c780a-396f-41de-afa9-4dd054942a04
+related:
+    - id: cc9cbe82-7bc0-4ef5-bc23-bbfb83947be7
+      type: derived
+status: experimental
+description: Detects potential malicious decoding of files using certutil
+references:
+    - https://attack.mitre.org/techniques/T1140/
+    - https://ezprotect.io/what-are-high-risk-file-types-and-how-to-block-users-from-accessing-them-in-salesforce/?
+    - https://cloud.google.com/blog/topics/threat-intelligence/targeted-attack-in-middle-east-by-apt34/
+    - https://lolbas-project.github.io/lolbas/Binaries/Certutil/
+    - https://news.sophos.com/en-us/2021/04/13/compromised-exchange-server-hosting-cryptojacker-targeting-other-exchange-servers/
+author: Peter Daniel
+date: 2025-02-04
+tags:
+    - attack.defense-evasion
+    - attack.t1140
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection_img:
+        - Image|endswith: '\certutil.exe'
+        - OriginalFileName: 'CertUtil.exe'
+    selection_cli:
+        CommandLine|contains:
+            - '-decode'
+            - '/decode'
+            - '-decodehex'
+            - '/decodehex'
+    selection_extensions:
+        CommandLine|contains:
+              # Program files
+              - '.exe'
```

Comment:
exe is probably matched with certutil.exe, if it is called with .exe. So we sadly need to remove it.

```suggestion
```